guides/source/active_support_core_extensions.md

```diff
@@ -2895,6 +2895,23 @@ Active Support extends these methods so that the argument may be another range i
 
 NOTE: Defined in `active_support/core_ext/range/compare_range.rb`.
 
+### Inclusion in a date time range
```

Comment:
This guide is to talk about extensions to core classes, since we are removing an extension we should not document it anymore here.

Reply:
I have updated the PR to address this feedback.